doc/index.rst

```diff
@@ -430,29 +467,55 @@ Maintainers of client repos are responsible for applying the changes that happen
 Run pre/post-process custom commands
 -----------------------------------------
 
-You may want to add custom steps prior to the setup defined in `./travis.sh <./travis.sh>`_. Example usecases:
+You may want to add custom steps prior/subsequent to the setup defined in `industrial_ci`. Example usecases:
 
 * A device driver package X in your repository or in your repository's dependency requires a prorietary library installed. This library is publicly available, but not via apt or any package management system and thus the only way you can install it is in a classic way (unzip, run installer etc.) (`More discussion <https://github.com/ros-industrial/industrial_ci/issues/14>`_).
 
 * You want to run `ros_lint` (`thi discussion <https://github.com/ros-industrial/industrial_ci/issues/58#issuecomment-223601916>`_ may be of your interest).
 
-In such cases, you can specify the script(s) in `BEFORE_SCRIPT` and/or `AFTER_SCRIPT` variables. For example::
+Customize within the CI process
+++++++++++++++++++++++++++++++++
+
+If what you want to customize is within the `CI process <#what-are-checked>`_, you can specify the script(s) in `BEFORE_SCRIPT` and/or `AFTER_SCRIPT` variables. For example::
 
   env:
     global:
       - BEFORE_SCRIPT='./your_custom_PREprocess.sh'
       - AFTER_SCRIPT='./your_custom_POSTprocess.sh'
   script:
-    - .ci_config/travis.sh
+    - .ci_config/ci.sh
+
+Multiple commands can be passed, as in a general `bash` manner. Using semi-colon as a delimitter,::
```

Comment:
I would not recommend semi-colons, these are error-prone in this context.

Reply:
What's better if you want to pass multiple commands? `&&` and/or `||`?

Reply:
Ok I got it https://github.com/ros-industrial/industrial_ci/pull/234/files#r159641174

Reply:
For the limited scope of `BEFORE_SCRIPT` using `&&` should be enough.
More complicated things should go into a script file.